manifests/metallb.yaml

```diff
@@ -296,7 +338,7 @@ spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
       serviceAccountName: speaker
-      terminationGracePeriodSeconds: 0
+      terminationGracePeriodSeconds: 2
```

Comment:
Curious why the change here?

Reply:
It's to give it the time to exit properly / leave the cluster
I chose 1sec max to leave the cluster so 2s for the clean shutdown